bootstrap_cfn/fab_tasks.py

```diff
@@ -496,6 +494,38 @@ def get_zone_id():
                 "for zone name '%s'...", zone_id, zone_name)
     return zone_id
 
+def get_hosted_name():
+    cfn_config = get_config()
+    r53_conn = get_connection(R53)
+    try:
+        for elb in cfn_config.data['elb']:
+            try:
+                record = "{name}.{hosted_zone}".format(**elb)
+                zone_name = elb['hosted_zone']
+            except KeyError:
+                raise CfnConfigError("No hosted_zone in yaml, unable to find hosted zone")
+            logger.info("fab_tasks::get_hosted_name: Found hosted zone name '%s' in config...", elb['hosted_zone'])
+            if zone_name.endswith('.'): 
+                return zone_name[:-1]
+            else:
+                return zone_name
```

Comment:
for multiple ELBs, this returns the first elb['hosted_zone']. I think we may want to return specific one that matches elb['name']